superset-frontend/src/dashboard/util/newEntitiesFromDrop.js

```diff
@@ -51,8 +51,8 @@ export default function newEntitiesFromDrop({ dropResult, layout }) {
     rowWrapper.children = [newDropChild.id];
     rowWrapper.parents = (dropEntity.parents || []).concat(dropEntity.id);
     newEntities[rowWrapper.id] = rowWrapper;
-    newDropChild = rowWrapper;
     newDropChild.parents = rowWrapper.parents.concat(rowWrapper.id);
+    newDropChild = rowWrapper;
```

Comment:
The intent here was clearly to add the wrapper as the parent to the wrapped component - due to the incorrect order here the wrapper's id was incorrectly added to the wrapper's parents, not the wrapped component's ids.